diff --git a/node_modules/@strapi/plugin-upload/admin/src/components/UploadAssetDialog/AddAssetStep/AddAssetStep.js b/node_modules/@strapi/plugin-upload/admin/src/components/UploadAssetDialog/AddAssetStep/AddAssetStep.js
index 07de8e1..e5342cf 100644
--- a/node_modules/@strapi/plugin-upload/admin/src/components/UploadAssetDialog/AddAssetStep/AddAssetStep.js
+++ b/node_modules/@strapi/plugin-upload/admin/src/components/UploadAssetDialog/AddAssetStep/AddAssetStep.js
@@ -18,6 +18,7 @@ import getTrad from '../../../utils/getTrad';
 
 import { FromComputerForm } from './FromComputerForm';
 import { FromUrlForm } from './FromUrlForm';
+import { FromAiForm } from 'strapi-ai-image-plugin/admin/src/components/Upload/FromAiForm';
 
 export const AddAssetStep = ({ onClose, onAddAsset, trackedLocation }) => {
   const { formatMessage } = useIntl();
@@ -54,6 +55,12 @@ export const AddAssetStep = ({ onClose, onAddAsset, trackedLocation }) => {
                 defaultMessage: 'From URL',
               })}
             </Tab>
+            <Tab>
+              {formatMessage({
+                id: getTrad('modal.nav.ai'),
+                defaultMessage: 'AI generation',
+              })}
+            </Tab>
           </Tabs>
 
           <Divider />
@@ -73,6 +80,13 @@ export const AddAssetStep = ({ onClose, onAddAsset, trackedLocation }) => {
               trackedLocation={trackedLocation}
             />
           </TabPanel>
+          <TabPanel>
+            <FromAiForm
+              onClose={onClose}
+              onAddAsset={onAddAsset}
+              trackedLocation={trackedLocation}
+            />
+          </TabPanel>
         </TabPanels>
       </TabGroup>
     </>
